src/OVGA.cpp

```diff
@@ -381,6 +381,7 @@ void Vga::handle_messages()
       case SDL_MOUSEWHEEL:
           scroll_x = event.wheel.x;
           scroll_y = event.wheel.y;
+          set_window_grab(WINGRAB_FORCE);
```

Comment:
The purpose of WINGRAB_FORCE is to allow for click and drag operations confined to a region (grab on mouse down, restore to user preference on mouse up). When you do this here, it will force mouse grab on permanently.